tests/specs/test-access-control.js

```diff
@@ -56,9 +57,9 @@ describe('Access Control Page - Verify that', function() {
 
         });
         accessControlPage.submit_token(valid_token);
-        loginpage = browser.element('h2=Installed snaps');
-        loginpage.waitForVisible();
-        expect(loginpage.getText(), "Login Failed with valid token").to.contain('Installed snaps');
+        var snaplist = homePage.installedsnaps;
+        snaplist.waitForVisible();
+        assert.isNotNull(snaplist);
```

Comment:
isNotNull on json object would always be true, I think keeping the existing assertion would be more appropriate, so it should be like
var snaplist = homePage.installedsnaps;
 snaplist.waitForVisible();
 expect(snaplist.getText(), "Login Failed with valid token").to.contain('Installed snaps');